lib/resources/r4/immunization.yaml

```diff
@@ -133,8 +133,8 @@ fields:
     }
   note: |
     <ul>
-      <li> encounter is required field to create charted immunizations </li>
-      <li> encounter is not a required field to create staged immunizations. </li>
+      <li> Encounter is required field to create charted immunizations. </li>
```

Comment:
since this is referring to the field name, I'd rather not capitalize this so it still matches. `resourceType` above has a similar note, so however we have it, they should be consistent.

Reply:
That's fair.
Matching the parameter is the correct route.